combine_libraries.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+import argparse
+from pathlib import Path
+import shutil
+import sys
+
+import numpy as np
+import openmc.data
+
+
+def library_in_list(lib, lib_list):
+    """Returns whether the given nuclide library is present in the given list of libraries.
+
+    Does this by checking whether libraries have the same type and material list.
+    """
+    for lib_in_list in lib_list:
+        if isinstance(lib_in_list, type(lib)):
+            if lib['type'] == lib_in_list['type'] \
+               and np.array_equal(lib['materials'], lib_in_list['materials']):
+                return True
+    return False
+
+
+description = """
+Script to combine nuclide files from multiple OpenMC HDF5 libraries into a single library.
+
+"""
+class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
+                      argparse.RawDescriptionHelpFormatter):
+    pass
+
+parser = argparse.ArgumentParser(
+    description=description,
+    formatter_class=CustomFormatter
+)
+parser.add_argument('-d', '--destination', type=Path,
+                    help='Directory to create new library in', required=True)
+parser.add_argument('-l', '--libraries', type=Path,
+                    help='List of data libraries to create', required=True, nargs='+')
+parser.add_argument('-n', '--no-copy', dest='copy', action='store_true',
+                    help='Don\'t copy library files, just create a cross_sections.xml file')
```

Comment:
Using double quotes would allow you to avoid escaping the internal single quote:
```suggestion
                    help="Don't copy library files, just create a cross_sections.xml file")
```